critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -352,7 +353,12 @@ private void doSync() {
         HttpPost post = new HttpPost(transferURL, content, true);
         post.setConnectionTimeout(6000); // ms
         post.setReadTimeout(60000);      // ms
-        post.prepare();
+        try {
+            post.prepare();
+        } catch (RangeNotSatisfiableException unexpected) {
```

Comment:
this local catch and wrap in RuntimeException is not the approp style for this class. I know it won't happen in practice, but say there is a poorly behaved file service that throws a 416... this will cause the while FileSyncJob to terminate rather than just use one of the retry strategies. So this should just be thrown here and dealt with in the approp catch up in doSync. 

In practice, doSync will just abort the job (`catch(Exception ex)` on line 202) and this wrap in RuntimeException will just hide the real cause.

You also need to add this exception to the approp catch in syncArtifact; there I think the catch on line 430 is the right choice: log at debug and remove the offending URL. As it is now, the RangeNotSatisfiableException will be rethrown  at line~446 and cause the job to terminate.

This probably seems pedantic since FileSyncJob doesn't curently do range requests (it is going to in the near future), but the main thing is to classify failures and be able to easily change the reaction as we gradually improve this code... and the wrap-in-RuntimeException will make it hard to deal with this exception correctly.